app/server.py

```diff
@@ -77,6 +77,7 @@ def index():
 @app.route('/inventory', methods=[GET])
 def query_prod_info():
     """ Query specific entries in the Inventory system """
+    app.logger.info("GET received, List all.")
```

Comment:
This route also responses to the query API. Please also log the specific query conditions

Reply:
Addressed.
<img width="559" alt="screen shot 2018-05-01 at 11 16 30 am" src="https://user-images.githubusercontent.com/19756129/39478786-5bdbd09e-4d31-11e8-8eb2-66a761875a31.png">